core/templates/dev/head/services/RteHelperService.js

```diff
@@ -154,6 +154,7 @@ oppia.factory('RteHelperService', [
           return rte;
         }
 
+
```

Comment:
Drop this newline. In general please check the "Files Changed" tab and remove spurious changes.

Reply:
Sorry, I didn't notice this, fixed it.